konami/identifiers.md

```diff
@@ -0,0 +1,131 @@
+# Konami Identifiers
+
+This file describes the various identifiers and their formats that are used in products by Konami.
+
+## Sources
+
+* [bemaniwiki](http://bemaniwiki.com/)
+* Game/cabinet manuals
+* MAME
+* Various sources around the internet
+
+## Format
+
+The format of each identifier/ID is described as a perl/php/shell-style string with `$` denoting
+that a variable is resolved, e.g. `$a` "resolve variable a". Additionally, a more complex regex
+string is provided to constrain the format properly.
+
+### Variables and values
+
+The following variables are used across the different identifiers and can denote the listed values:
+
+* `$p`: Product
+  * `G`: Game
+* `$t`: Type of product
+  * `C`: Upgrade: software only
+  * `E`: Upgrade: hardware (conversion) kit + software
+  * `K`: Upgrade: offline/no eamuse software version
+  * `N`: ???
+  * `Q`: Dedicated cabinet assembly/main body
+  * `U`: "Utilities", e.g. DVD software update kit, replacement monitors
+  * `X`: Hardware/board ???
```

Comment:
Let's add a few question marks here since I don't think at this point anything is sure. Also, we found `O` the other day.

Reply:
Since there are likely many spots with best guessing or not knowing stuff, it might be useful to define a tag that allows us to find these spots easily using a simple text search. What about `[???]` = unknown meaning or when following some text meaning guess?